### PR TITLE
Update setup-eks.md

### PR DESCRIPTION
**Pull request by rochabr**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [X ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ X] New file and folder names are globally unique
- [ X] Page references use shortcodes instead of markdown or URL links
- [ X] Images use HTML style and have alternative text
- [ X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Updating docs with instructions on how to install Dapr 1.14 on EKS.

### CODE REVIEW

File: daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md [hunk 1]
```diff
@@ -25,20 +26,57 @@ This guide walks you through installing an Elastic Kubernetes Service (EKS) clus
...
+      - name: mng-od-4vcpu-8gb
+        desiredCapacity: 2
+        minSize: 1
+        maxSize: 5
+        instanceType: c5.xlarge
+        privateNetworking: true
+    
+    addons:
+      - name: vpc-cni # no version is specified so it deploys the default version
+        attachPolicyARNs:
```
Comment (yaron2) [inline daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md:55]: Do we need the commented section here?

Comment (rochabr) [reply daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md:55]: removed

File: daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md [hunk 2]
```diff
@@ -49,11 +87,37 @@ This guide walks you through installing an Elastic Kubernetes Service (EKS) clus
    --source-group [your_security_group]
    ```
 
+2. Dapr 1.14 Scheduler service requires a default storage class to be set. If you don't have one, patch your cluster with the command below:
```
Comment (yaron2) [inline daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md]: Suggest to change `Dapr 1.14 Scheduler service requires a default storage class to be set` to 'Add a default storage class if you don't have one:`.

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md (hunk 2 above)
Comment (rochabr) [reply daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md]: Done

**Review by yaron2 — APPROVED** [review]

**Pull request merged**